sample-kotlin/build.gradle

```diff
@@ -25,7 +25,6 @@ android {
     }
   }
   sourceSets {
-    main.java.srcDirs += 'src/main/kotlin'
```

Comment:
You have right,  strange thing, I don't change this line... Perhaps in android studio.
I roll back this line.